src/Microsoft.Health.Fhir.Liquid.Converter/Hl7v2/Models/UnusedHl7v2Field.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.Hl7v2.Models
+{
+    public class UnusedHl7v2Field
```

Comment:
>UnusedHl7v2Field [](start = 17, length = 16)

possible to provide the position of the field?

Reply:
Yes, provided in the output. One example output could be found [here](https://microsoftapc-my.sharepoint.com/:u:/g/personal/boywu_microsoft_com/EQrqXvlERhhKuECMr99KEDQBNeyn69WepGC0v-gFI6A63Q?e=RdnY8y).

Reply:
Simplified output TraceInfo in above link.